src/components/UI/Form.styled.js

```diff
@@ -50,6 +52,49 @@ export const Label = styled.label`
 	padding: 0.2rem;
 	justify-content: center;
 	font-size: 2rem;
-
 	font-weight: bolder;
+	margin-top: 20px;
+`;
+
+export const WrapperList = styled.ul`
+	display: flex;
+
+	list-style-type: none;
+	flex-wrap: wrap;
+	background-color: black;
+	width: 30rem;
+	margin: 25px;
+	gap: 10px;
+`;
+
+export const ContainerSearch = styled.div`
+	display: flex;
+	align-items: center;
+	border: solid palevioletred;
```

Comment:
Does this work? You might need the border-width, too: 
```suggestion
	border: 2px solid palevioletred;
```

Reply:
I changed that